Add missing file format (#3480)
qgis/QGIS-Documentation: source/docs/pyqgis_developer_cookbook/vector.rst

=====================

You can write vector layer files using :class:`QgsVectorFileWriter` class. It
supports the vector formats that OGR supports (GeoPackage, Shapefile, GeoJSON,
KML and many more).

There are two ways of exporting a vector layer:

* from an instance of :class:`QgsVectorLayer`

  .. code-block:: python

    # Write to a GeoPackage (default)

>     error = QgsVectorFileWriter.writeAsVectorFormat(layer, "my_data", "")

Reviewer: ```suggestion
    error = QgsVectorFileWriter.writeAsVectorFormat(layer, "/path/to/folder/my_data", "")
```

    if error[0] == QgsVectorFileWriter.NoError:
        print("success!")

    # Write to an ESRI Shapefile format dataset using UTF-8 text encoding

>     error = QgsVectorFileWriter.writeAsVectorFormat(layer, "my_esridata", "UTF-8",

Reviewer: ```suggestion
    error = QgsVectorFileWriter.writeAsVectorFormat(layer, "/path/to/folder/my_esridata", "UTF-8",
```

                                                    driverName="ESRI Shapefile")
    if error[0] == QgsVectorFileWriter.NoError:
        print("success again!")

  The third (mandatory) parameter specifies output text encoding.
  Only some drivers need this for correct operation - Shapefile is one of them

>   (other drivers will ignore this parameter)

Reviewer: ```suggestion
  (other drivers will ignore this parameter).
```

>   --- if you are not using international characters you do not have to worry about

Reviewer: ```suggestion
  Specifying the correct encoding is important if you are using international (non US-ASCII) characters.
```

>   the encoding. 

Reviewer: ```suggestion
```
